x-pack/plugins/maps/common/descriptor_types/data_request_descriptor_types.ts

```diff
@@ -19,17 +19,19 @@ export type Timeslice = {
 };
 
 // Global map state passed to every layer.
-export type MapFilters = {
+export type DataFilters = {
   buffer?: MapExtent; // extent with additional buffer
   extent?: MapExtent; // map viewport
   filters: Filter[];
-  query?: MapQuery;
+  query?: Query;
   refreshTimerLastTriggeredAt?: string;
   searchSessionId?: string;
   timeFilters: TimeRange;
   timeslice?: Timeslice;
   zoom: number;
   isReadOnly: boolean;
+  searchSessionMapBuffer?: MapExtent;
```

Comment:
Why is searchSessionMapBuffer added to DataFilters? How does this PR change how searchSessionMapBuffer is used?

Reply:
this is wrong. It's a left-over from an early change. I added this property because I had annotated the prev/nextQuery-objects as `DataFilters` initially. I removed that type annotation as they are not _really_ data-filters rather just temp-objects that are used for the `setQuery` action. 

This PR does not introduce `searchSessionMapBuffer` elsewhere, so I'll just remove it here.